conf/40-escape

```diff
@@ -2,6 +2,7 @@
 # Make zsh exit on escape
 # © 2013 Axel Wagner and contributors (see also: LICENSE)
 function _shellex_exit {
+    rm $SHELLEX_SIZE_FILE
```

Comment:
This really doesn't belong here. What about if we unlink() it from the LD_PRELOAD lib after successfully reading from it?